src/analysis/normed_space/basic.lean

```diff
@@ -374,6 +374,11 @@ instance submodule.normed_group {𝕜 : Type*} {_ : ring 𝕜}
 { norm := λx, norm (x : E),
   dist_eq := λx y, dist_eq_norm (x : E) (y : E) }
 
+@[simp] lemma coe_norm {𝕜 : Type*} {_ : ring 𝕜}
```

Comment:
```suggestion
/-- If `x` is an element of a submodule `s` of a normed group `E`, its norm in `s` is equal to its
norm in `E`.

See note [implicit instance arguments]. -/
@[simp] lemma coe_norm {𝕜 : Type*} {_ : ring 𝕜}
```
Could you also replace the last sentence of the docstring of `submodule.normed_group` with the link to this library note?